ckan/templates/home/snippets/search.html

```diff
@@ -1,4 +1,4 @@
-{% set tags = h.get_facet_items_dict('tags', c.search_facets, limit=3) %}
+{% set tags = h.get_facet_items_dict('tags', search_facets, limit=3) %}
```

Comment:
FYI This always returns no tags because `search_facets` does not exist because it is not being set in the `extra_vars` object passed to this template from the blueprint `home.index`.

Reply:
Thanks @MarkCalvert ! Would you like to create a PR fixing it? :)

Reply:
Sure. We will add it to our list to contribute back a PR.

Reply:
This was fixed in 2.10 dev branch but i didnt added the labels for backports 
https://github.com/ckan/ckan/issues/7519